yt_dlp/postprocessor/embedthumbnail.py

```diff
@@ -102,7 +100,7 @@ def run(self, info):
         elif info['ext'] in ['mkv', 'mka']:
             options = list(self.stream_copy_opts())
 
-            mimetype = 'image/%s' % ('png' if thumbnail_ext == 'png' else 'jpeg')
+            mimetype = 'image/%s' % ('jpeg' if thumbnail_ext in ('jpg', 'jpeg') else thumbnail_ext)
```

Comment:
re-thinking this, it could also be simplified to
```suggestion
            mimetype = f'image/{thumbnail_ext.replace("jpg", "jpeg")}'
```